### PR TITLE
feat(landing): Updates qiskit definition

### PR DESCRIPTION
**Pull request by y4izus**
## Changes

Part of https://github.com/Qiskit/qiskit.org/issues/3125

Updates Qiskit definition

## Implementation details

We need to replace the current Qiskit definition with this one: _Qiskit {{ qiskitPronunciation }} is an open-source SDK for working with quantum computers focus in how build circuits and optimize them using the last transpiler passes, and bringing access to the last quantum hardware interfaces._

## How to read this PR

Go to the landing page and make sure that the description is correct

### CODE REVIEW

**Review by eddybrando — CHANGES_REQUESTED** [review]
I believe that we need to review the copy.

**Comment by y4izus** [comment]
Waiting for the final text, I put the PR on ON-HOLD

**Comment by techtolentino** [comment]
Quick note: once you have the final definition, we might want to update it in the site `meta description`, which I believe pulls from [package.json](https://github.com/Qiskit/qiskit.org/blob/yg-updates-qiskit-definition/package.json#L4)

**Comment by 1ucian0** [comment]
xlg-1312px-16 column-3

File: components/Landing/HeroMoment/LandingHeroMomentComponent.vue [hunk 1]
```diff
@@ -2,37 +2,65 @@
...
           class="hero-moment__container__image"
-          src="/images/landing-page/new-hero-illustration.png"
+          src="/images/landing-page/cryo.png"
           alt="A visual composition of a sketched a quantum computer backdrop, a progress bar indicating a running job, and a laptop screen displaying the text 'running job...'"
         />
+        <div class="hero-moment__container__laptop">
+          <img
+            class="hero-moment__container__laptop__frame"
+            src="/images/landing-page/laptop.png"
+            alt="Laptop"
```
Comment (y4izus) [inline components/Landing/HeroMoment/LandingHeroMomentComponent.vue:50]: Could you add more description to the `alt`? maybe we can reference to the code too

**Review by y4izus — COMMENTED** [review]
Could you take a look at my comments? I didn't have time to finish all the code review, so @eddybrando please, could you review it too?

**Comment by eddybrando** [comment]
This is design file: https://www.figma.com/file/CM5H3pOzE1CKnF0WoMYFZQ/3329-Utility-messaging?type=design&node-id=222-7829&mode=design&t=TGPYVp6UKABoZlXp-0

File: components/Landing/HeroMoment/LandingHeroMomentComponent.vue [hunk 2]
```diff
@@ -94,23 +137,68 @@ const qiskitPronunciation = Math.random() < 0.5 ? "[kiss-kit]" : "[quiss-kit]";
...
       z-index: -1;
 
-      @include carbon.breakpoint-down(md) {
+      @include carbon.breakpoint-down(lg) {
         display: none;
       }
     }
+
+    &__laptop {
+      width: 793px;
```
Comment (eddybrando) [inline components/Landing/HeroMoment/LandingHeroMomentComponent.vue:157]: We usually don't work with such specific sizes but rather proportional, responsive sizes. Is there a need to do it like this in this specific case? This also goes for the other very specific size definitions in these styles.

File: components/Landing/HeroMoment/LandingHeroMomentComponent.vue [hunk 3]
```diff
@@ -123,17 +211,31 @@ const qiskitPronunciation = Math.random() < 0.5 ? "[kiss-kit]" : "[quiss-kit]";
 
   &__title {
     margin-bottom: carbon.$spacing-06;
+    font-weight: 500;
+    font-size: 5rem;
```
Comment (eddybrando) [inline components/Landing/HeroMoment/LandingHeroMomentComponent.vue:215]: We should use a font definition from the reusable defined variables that we get from the Carbon library, like we do in the rest of the codebase.

File: components/Landing/HeroMoment/LandingHeroMomentComponent.vue [hunk 4]
```diff
@@ -123,17 +211,31 @@ const qiskitPronunciation = Math.random() < 0.5 ? "[kiss-kit]" : "[quiss-kit]";
...
+    display: flex;
+    align-items: center;
 
     // TODO: Force pointer events to allow the user to select text. Remove
     // when decomissioning the Metal page.
     pointer-events: auto;
 
     @include carbon.breakpoint-up(md) {
       margin-bottom: carbon.$spacing-07;
+      font-size: 7rem;
```
Comment (eddybrando) [inline components/Landing/HeroMoment/LandingHeroMomentComponent.vue:225]: We should use a font definition from the reusable defined variables that we get from the Carbon library, like we do in the rest of the codebase.

File: components/Landing/HeroMoment/LandingHeroMomentComponent.vue [hunk 5]
```diff
@@ -144,19 +246,35 @@ const qiskitPronunciation = Math.random() < 0.5 ? "[kiss-kit]" : "[quiss-kit]";
     // when decomissioning the Metal page.
     pointer-events: auto;
 
+    &__title {
+      color: carbon.$gray-80;
+    }
+
+    &__divider {
+      border: 1px solid carbon.$gray-80;
+      border-width: 1px 0 0;
```
Comment (eddybrando) [inline components/Landing/HeroMoment/LandingHeroMomentComponent.vue:255]: We're defining border-width twice.

File: components/Landing/HeroMoment/LandingHeroMomentComponent.vue [hunk 6]
```diff
@@ -2,37 +2,69 @@
...
     <LandingHeroMomentGrid>
       <div class="cds--grid hero-moment__container">
-        <LandingHeroMomentVersionInfo
-          class="hero-moment__version-info"
-          :version="version"
-        />
         <div class="cds--row">
           <h1
-            class="cds--col-md-3 cds--col-lg-5 cds--col-xlg-6 cds--col-max-7 hero-moment__title"
+            class="cds--col-md-4 cds--col-lg-5 cds--col-xlg-5 cds--col-max-7 hero-moment__title"
```
Comment (techtolentino) [inline components/Landing/HeroMoment/LandingHeroMomentComponent.vue:7]: I don't normally see the carbon grid classes being applied to a text element. Normally, they're classes of a surrounding `div`. 

I'd suggest

```html
<div class="cds--col-md-4 cds--col-lg-5 cds--col-max-7">
  <h1 class="hero-moment__title"></h1>
</div>
```

File: components/Landing/HeroMoment/LandingHeroMomentComponent.vue [hunk 7]
```diff
@@ -2,37 +2,69 @@
...
+              src="/images/landing-page/logo_wordmark.svg"
+              alt="The Qiskit logo"
+            />
           </h1>
         </div>
         <div class="cds--row">
-          <p
-            class="cds--col-sm cds--col-md-3 cds--col-lg-5 cds--col-xlg-4 hero-moment__description"
+          <div
+            class="cds--col-sm cds--col-md-4 cds--col-lg-5 cds--col-xlg-5 hero-moment__description"
```
Comment (techtolentino) [inline components/Landing/HeroMoment/LandingHeroMomentComponent.vue:19]: if we have `cds--col-lg-5`, do we also need `cds--col-xlg-5`?

**Review by techtolentino — CHANGES_REQUESTED** [review]
Left a few comments... it's coming along nicely

Thanks @paaragon

**Comment by y4izus** [comment]
Closing this is favor of https://github.com/Qiskit/qiskit.org/pull/3396

**Pull request closed without merging**